### ISSUE TITLE
ResizeSensor doesn't - actually - support native collections

### ISSUE THREAD

**Issue by rasmusfl0e** [marcj/css-element-queries#21]
From [Line 131](https://github.com/marcj/css-element-queries/blob/master/src/ResizeSensor.js#L131) of ResizeSensor.js:

```
        if ('array' === typeof element
```

Firstly - native element collections are not Arrays - but NodeLists or HTMLCollections.
Secondly - `typeof` doesn't return `'array'` - even for arrays; it returns `'object'` - sadly. And same goes for NodeLists and HTMLCollections... 

... which all means that that line of code doesn't actually do anything ;)

Something like the following should work:
```
        var elementString = element.toString();
        if (elementString === "[object NodeList]" || elementString === "[object HTMLCollection]"
```

**Comment by marcj** [comment]
> Did you guys ever test this with document.querySelectorAll?

Obviously not. Feel free to test and fix it.

**Comment by rasmusfl0e** [comment]
Looks like this was fixed in 5127e931ca16a2300d605f39546c9325c3023bf0 .
I'll just close this then ;)
